pkg/interface/src/views/apps/chat/components/lib/chat-window.tsx

```diff
@@ -250,7 +253,7 @@ export class ChatWindow extends Component<ChatWindowProps, ChatWindowState> {
         {messages.length ? <VirtualList
           ref={this.virtualList}
           style={{height: '100%', width: '100%', visibility: this.state.initialized ? 'initial' : 'hidden'}}
-          totalCount={mailboxSize}
+          totalCount={mailboxSize + stationPendingMessages.length}
```

Comment:
Is this being passed to the messages inside? The last non-pending message has `pb3` when the pending should have `pb3`.

Reply:
good point